core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java

```diff
@@ -95,59 +92,63 @@ protected Color4f getDefaultMarkerColor() {
 
   @Override
   protected void createVisuals() {
+    initialize();
+    sgVisuals = new Visual[] {createLensVisual(),
+        createCylinderVisual("Camera Cylinder 1", RADIUS),
+        createCylinderVisual("Camera Cylinder 2", RADIUS * 3),
+        createBoxVisual()};
 
-    this.sgAppearance = new SimpleAppearance();
-    this.sgAppearances = new SimpleAppearance[] {sgAppearance};
+    addLaserSightLine();
+    updateViewGeometry();
```

Comment:
assuming you meant to drop setViewingAngle here?

Reply:
Yes. Those values are now set in initialize() and don't need to be set again.

Reply:
Deleted it and a few related lines.